dockers/docker-sonic-telemetry/telemetry.sh

```diff
@@ -60,6 +60,8 @@ TELEMETRY_ARGS+=" --port $PORT"
 CLIENT_AUTH=$(echo $GNMI | jq -r '.client_auth')
 if [ -z $CLIENT_AUTH ] || [ $CLIENT_AUTH == "false" ]; then
     TELEMETRY_ARGS+=" --allow_no_client_auth"
+else
+    TELEMETRY_ARGS+=" --client_auth $CLIENT_AUTH"
```

Comment:
https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-telemetry.yang#L57
client_auth is a flag for requiring client auth, but you are using it as client auth type？

Reply:
Closing this PR. And will open a new PR addressing the comments

Reply:
If you want to address PR comments, better to keep this PR open, and push new commit in the same PR. So the comment will have enough context and easy to track.

Reply:
Fine @qiluo-msft

Reply:
Client Authentication Argument in Telemetry dialin server (https://github.com/sonic-net/sonic-gnmi/blob/master/telemetry/telemetry.go) accepts jwt,password,certification. Made changes in telemetry.sh script to accept the different client authentication modes.